modules/FindMagnumExtras.cmake

```diff
@@ -79,7 +81,13 @@ set(_MAGNUMEXTRAS_MAGNUM_DEPENDENCIES )
 foreach(_component ${MagnumExtras_FIND_COMPONENTS})
     string(TOUPPER ${_component} _COMPONENT)
 
-    # (none yet)
+    if(_component MATCHES Octree)
+        set(_MAGNUMEXTRAS_${_COMPONENT}_MAGNUM_DEPENDENCIES Shapes)
+    endif()
+
+    if(_component MATCHES SceneGraph)
+        set(_MAGNUMEXTRAS_${_COMPONENT}_MAGNUM_DEPENDENCIES SceneGraph)
+    endif()
```

Comment:
I'm not sure how the SceneGraph component of extras will interact with SceneGraph component of Magnum, so please just merge all that into the Octree library as I said in the previous review.